extensions/wikia/UserLogin/css/UserLogin.scss

```diff
@@ -75,3 +75,7 @@ $input-width: 230px;
 		display: none;
 	}
 }
+
+.hidden {
```

Comment:
Yup, it's a great use case of !important and it should be there to override any stronger selectors / inline styles. @armonr :+1: